oaknational/Oak-Web-Application
# Fix/signed unsigned video fixes

## Description

- Updated copy
- Fixed missing thumbnails and .vtt files
- Updated loader to 'Loading' text under video dropdown so the loader of video doesn't prevent user from accessing other part of the page while videos are loading

## Issue(s)

Fixes #1130

## How to test

1. Go to /beta/teachers/lessons/macbeth-lesson-1
2. Click on Video dropdown
3. You should see loading text while videos are loading and then be able to switch between signed language and normal video without a flicker

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
